tests/acceptance/features/webUIFiles/fileFolderActionMenu.feature

```diff
@@ -0,0 +1,38 @@
+Feature: User can see the file or folder actions menu options
+  As a user
+  I want to be able to see the actions menu options of file or folder
+  So that the menu options of the selected file type or folder are visible to me
+
+Background: prepare user and files
+    Given user "user1" has been created with default attributes
+    And user "user1" has logged in using the webUI
+    And the user has browsed to the files page
+    
+  Scenario: observe different actions menu options on selecting different file types or folder
+    Given user "user1" has uploaded file with content "pdf file" to "lorem.pdf"
+    And the user has reloaded the current page of the webUI
+    And the app-sidebar for file "lorem.txt" is visible on the webUI
```

Comment:
```suggestion
     ... has been visible
```
Given steps are written in the past perfect tense

Reply:
~~`past perfect`~~ `present perfect`

Reply:
true :smile: